src/components/BlockChain.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect, ReactElement } from 'react';
```

Comment:
you shouldn't create a new branch for this, delete this branch and rebase the other one.
Also the CI on this branch isn't green